west.yml

```diff
@@ -274,7 +274,7 @@ manifest:
         - hal
     - name: hostap
       path: modules/lib/hostap
-      revision: 697fd2cf5cbbd0c5375fc34761b6a9d7489a67d2
+      revision: 8499c3195b3d3815417d162c69873a226262f25d
```

Comment:
commit title should be `manifest: hostap: Pull.....` and also, you need to link to the PR and not the specific commit, since the PR is not merged yet, the commit SHA will keep changing on every force push. `pull/80/head` should do it.